koji-prometheus-exporter.py

```diff
@@ -234,6 +243,11 @@ def only(tasks, states):
             yield task
 
 
+def getLastHostUpdate(host, ts):
```

Comment:
It's just a style thing, but can you make this `get_last_host_update(host, ts)` so that it matches the style of other functions in the module here?

Reply:
Yes, and then no. Function removed as part of multicallsession work.